packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -7,6 +7,7 @@ import { Agent as hsAgent, request as hsRequest, RequestOptions } from "https";
 import { NODEJS_TIMEOUT_ERROR_CODES } from "./constants";
 import { getTransformedHeaders } from "./get-transformed-headers";
 import { writeRequestBody } from "./write-request-body";
+import {setSocketKeepAlive, SocketKeepAliveOptions} from "./set-socket-keep-alive";
```

Comment:
`prettier` code formatter should be run on these files

Reply:
Done!